src/storage/src/row_serde/row_serde_util.rs

```diff
@@ -42,9 +42,3 @@ pub fn deserialize_pk_with_vnode(
     let pk = deserializer.deserialize(&key[VirtualNode::SIZE..])?;
```

Comment:
Maybe we can use `let key = TableKey(key)` `key.vnode_part()` `key.key_part()` here.

Reply:
This requires a bigger refactoring since the memtable key is in `Bytes` not `TableKey` type. Given that the memtable is already in storage, I will try to expose the `TableKey` struct in memtable as well in a separate PR and fix this issue then.